packages/svg-icons/src/index.ts

```diff
@@ -1 +1,30 @@
 export const test = 'test';
+
+export const carouselContents = [
```

Comment:
These icons are not ok. Everything inside `svg-icons` should be exported separately. This is mocked data and should be placed outside the packages folder.